client/components/NetworkList.vue

```diff
@@ -143,6 +298,111 @@ export default {
 				order: channel.network.channels.map((c) => c.id),
 			});
 		},
+		toggleSearch(event) {
+			event.preventDefault();
```

Comment:
change this to `return false`, mousetrap can do it.

and add a `textarea` check like in https://github.com/thelounge/thelounge/pull/3720

Reply:
This should be fixed now.